cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -454,14 +454,18 @@ pub type XcmRouter = WithUniqueTopic<(
 	// GlobalConsensus with a pausable flag, if the flag is set true then the Router is paused
 	PausableExporter<
 		crate::SnowbridgeSystemFrontend,
-		SovereignPaidRemoteExporter<
-			(
+		(
+			UnpaidRemoteExporter<
 				bridging::to_ethereum::EthereumNetworkExportTableV2,
+				XcmpQueue,
+				UniversalLocation,
+			>,
+			SovereignPaidRemoteExporter<
 				bridging::to_ethereum::EthereumNetworkExportTable,
-			),
-			XcmpQueue,
-			UniversalLocation,
-		>,
+				XcmpQueue,
+				UniversalLocation,
+			>,
```

Comment:
Since `UnpaidRemoteExporter` now supports attaching a fee, how about applying it to the Snowbridge V1 route as well?

That would eliminate the need to maintain a balance for AH’s sovereign account on BH.

@vgeddes @acatangiu WDYT?

Reply:
how long do you plan to support Snowbridge v1 for? might not be worth the change if its lifetime is limited anyway (lower risks of breaking something)

Reply:
As we discussed internally, it’s likely that we will run V1 along side with V2 for quite some time.

Btw, aside from changing this line 
https://github.com/paritytech/polkadot-sdk/blob/e723cfa2030586f968bb37dd094a61912573a13c/bridges/snowbridge/primitives/outbound-queue/src/v1/converter/mod.rs#L135 to set the fee to zero, I don’t see many changes required.

There might be some cleanup needed, but considering that our integration tests cover most cases, I’m not too concerned about the breaking things.

Reply:
A seperate PR for this: https://github.com/yrong/polkadot-sdk/pull/20

Reply:
Looks good! Thanks Ron.

Reply:
@acatangiu @bkontur I've merged https://github.com/yrong/polkadot-sdk/pull/20. Please take another look at your convenience.